lib/ansible/modules/web_infrastructure/ansible_tower/tower_inventory_source.py

```diff
@@ -0,0 +1,339 @@
+#!/usr/bin/python
+# coding: utf-8 -*-
+#
+# (c) 2018, Adrien Fleury <fleu42@gmail.com>
+# GNU General Public License v3.0+
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'status': ['preview'],
+                    'supported_by': 'community',
+                    'metadata_version': '1.1'}
+
+
+DOCUMENTATION = '''
+---
+module: tower_inventory_source
+author: "Adrien Fleury (@fleu42)"
+version_added: "2.6"
+short_description: create, update, or destroy Ansible Tower inventory source.
+description:
+    - Create, update, or destroy Ansible Tower inventories source. See
+      U(https://www.ansible.com/tower) for an overview.
+options:
+    name:
+      description:
+        - The name to use for the inventory source.
+      required: True
+    description:
+      description:
+        - The description to use for the inventory source.
+      required: False
+      default: null
+    inventory:
+      description:
+        - The inventory the source is linked to.
+      required: True
+    source:
+      description:
+        - Types of inventory source.
+      choices:
```

Comment:
Could you make the choices match this more up-to-date list:

https://github.com/ansible/awx/blob/devel/awx/main/models/inventory.py#L971

...by _adding_ `rhv` and `tower`?  `azure` support was recently removed, but let's leave it in for backwards compatibility with older Towers.

Reply:
The choices are now matching the up-to-date list from AWX.